.coderabbit.yaml

```
@@ -0,0 +1,9 @@
+# yaml-language-server: $schema=https://coderabbit.ai/integrations/schema.v2.json
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)